uk.co.compucorp.civicrm.hrleaveandabsences/js/hrleaveandabsences.form.manage_entitlements.js

```diff
@@ -29,6 +30,46 @@ CRM.HRLeaveAndAbsencesApp.Form.ManageEntitlements = (function($) {
     });
   };
 
+  /**
+   * Add event listeners to events triggered by elements of managed by this class
+   *
+   * @private
+   */
+  ManageEntitlements.prototype._addEventListeners = function() {
+    this._listElement.find('tbody > tr').on('click', this._onListRowClick.bind(this));
+  };
+
+  /**
+   * This is the event handler for when the user clicks on a row of the calculations
+   * list.
+   *
+   * It shows the user a popup with details of the selected calculation. Even if the
+   * proposed entitlement was overridden, we display the original calculation.
+   *
+   * @param event
+   * @private
+   */
```

Comment:
But with the function I would have **this** pointing to the object, right (unless I called it with bind or something like that)?

Anyway, most of this front-end code will be rewritten in the future, when applying the new layout. I just wanted to do something a little better than a bunch of code inside a $(document).ready() but not too complicated.